layouts/shortcodes/dbm-postgres-agent-config-examples.md

```diff
@@ -60,26 +60,27 @@ instances:
         - 'table:employees'
 ```
 
-### Monitoring multiple databases 
-To specify the name of the database to connect to, use `dbname`. If omitted, the default system `postgres` database is queried. Additionally, configure `dbstrict` to `true` if you only want to gather metrics from the database provided in the `dbname` option.
+### Monitoring multiple databases with DBM and the default integration
+To specify the name of the database to connect to, use `dbname`. If omitted, the default system `postgres` database is queried.
```

Comment:
might want to add some words here to explain why only one of the instances has `dbm: true` I have some words about this here https://datadoghq.atlassian.net/wiki/spaces/DM/pages/2645229741/PostgreSQL+FAQ#How-to-monitor-multiple-databases-with-dbm-&-the-default-pg-integration

this part is important for customers to understand 

> If you wish to collect the [generic postgres integration]<INSERT LINK HERE> metrics from logical databases, other than the default `postgres` db, you must list them each in their instance config. **Note:** you should list each logical db without setting `dbm: true`, and then add one final instance with `dbm:true` so that we don't "double monitor" the databases. This is because database-montoring collects metrics from global views like `pg_stat_statements` and `pg_stat_activity`.